components/backup/Cargo.toml

```diff
@@ -66,9 +67,11 @@ yatp = { git = "https://github.com/tikv/yatp.git", branch = "master" }
 tokio = { version = "1.5", features = ["rt-multi-thread"] }
 tokio-stream = "0.1"
 async-channel = "1.4"
+protobuf = { version = "2.8", features = ["bytes"] }
```

Comment:
Where it is used ?

Reply:
i don't remember why it's added, maybe depend on something during coding. Anyway, it's not used for now. I will remove it.